docs/reference/remap/to_timestamp.cue

```diff
@@ -0,0 +1,54 @@
+remap: functions: to_timestamp: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The value that is to be converted to a timestamp. If a string, must be a valid representation of a `timestamp`, and no `default` exists, an `ArgumentError` will be raised."
```

Comment:
Given that we want to keep the argument descriptions succinct, it almost feels like we need two properties; `description`  and `documentation` or something, one being only a few words long and the other being more free-form, but hidden by default behind some kind of collapsible button on the website.